roles/ceph-mon/templates/ceph-mon.service.j2

```diff
@@ -23,6 +23,7 @@ ExecStart=/usr/bin/docker run --rm --name %i --net=host \
    --net=host \
    {% endif -%}
    -e CEPH_DAEMON=MON \
+   -e MON_NAME={{ ceph_mon_name }} \
```

Comment:
You're right, I forgot about that, no I don't want to expose it further to the user. Although I think we should document the options available in ceph-docker. I'm closing this.